### DISCUSSION TITLE
error on start container reason of cmd

### DISCUSSION THREAD

**Discussion by ev11ccaatt** [moby/moby#46514] [Q&A]
i try to run demo
but get
`        "Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: \"echo\": executable file not found in $PATH: unknown"`

**Comment by ev11ccaatt** [comment]
"Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: \"/bin/sh\": stat /bin/bash: no such file or directory: unknown"

**Comment by thaJeztah** [comment]
That depends on the image you're running. The error is that the `echo` command or binary was not found in the image you're running as a container.

What demo are you running? Is that something from the docker documentation?

**Comment by ev11ccaatt** [comment]
sorry it'smycode wrong
